docs/studios/unity/unity-sdk/creating-bots/javascript/validation-bots.mdx

```diff
@@ -149,6 +149,29 @@ export async function startScenario(rg) {
 
 These methods are in addition to the APIs defined in the [PlayTest Bots section](./playtest-bots#variables).
 
+### `recordValidation(validationName: string, result: "PASS" | "FAIL" | "WARNING", options);`
+
+Records a validation result that can later be seen in the Validations Dashboard. 
+
+#### Arguments
+Argument Name | Type | Default | Description
+---|---|---|---|
+`validationName` | `string` | | The name or message of the test to show in the UI
+`result` | `"PASS"` \| `"FAIL"` \| `"WARNING"` | | The result of the test
+`options` | `object` | `{}` | Additional options for the validation
+`options.icon` | `string` | | The icon to show in the UI. Can be `clock`, `heart`, `key`, or `sparkles`
+
+#### Return Value
+No return value
```

Comment:
To match other void methods in the docs:
```suggestion
None
```



Outside of current scope: We should probably change all of these to "Return Type" rather than "Value" and `void`